bundle/config/loader/process_include.go

```diff
@@ -31,6 +163,13 @@ func (m *processInclude) Apply(_ context.Context, b *bundle.Bundle) diag.Diagnos
 	if diags.HasError() {
 		return diags
 	}
+
+	// Add any diagnostics associated with the file format.
+	diags = append(diags, validateFileFormat(this, m.relPath)...)
```

Comment:
This can pass `this.Value()` directly. These functions don't seem to access other fields.

Reply:
done